src/rewriter.coffee

```diff
@@ -321,7 +328,12 @@ exports.Rewriter = class Rewriter
       if tag is ':'
         # Go back to the (implicit) start of the object.
         s = switch
-          when @tag(i - 1) in EXPRESSION_END then start[1]
+          when @tag(i - 1) in EXPRESSION_END
```

Comment:
Then, we have to make sure that for an implicit object (whether an actual implicit object or a class body) starting with a static computed property eg `@[b]: ->`, the start of the implicit object comes *before* the `@`